### PR TITLE
ppl: place constrained groups with mirrored pins first

### PR DESCRIPTION
**Pull request by eder-matheus**

### CODE REVIEW

**Comment by eder-matheus** [comment]
@oharboe Could you try this branch? It should fix the last error you're having.

File: src/ppl/test/add_constraint12.tcl [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# gcd_nangate45 IO placement
+source "helpers.tcl"
+read_lef Nangate45/Nangate45.lef
+read_def gcd.def
+
+set_io_pin_constraint -pin_names {resp_msg[3] resp_msg[2] resp_msg[14] req_val} -region top:0-18
```
Comment (maliberty) [inline src/ppl/test/add_constraint12.tcl:6]: Does this cover the partially mirrored group case?

**Review by maliberty — APPROVED** [review]

File: src/ppl/test/add_constraint12.tcl (hunk 1 above)
Comment (eder-matheus) [reply src/ppl/test/add_constraint12.tcl:6]: I've added a new test covering it.

**Comment by oharboe** [comment]
> @oharboe Could you try this branch? It should fix the last error you're having.

I see a different failure. I'll articulate a test-case.

**Pull request merged**